src/dropzone/DropzoneDirective.ts

```diff
@@ -37,9 +37,22 @@ export class DropzoneDirective implements ng.IDirective {
       addRemoveLinks: true,
       dictCancelUpload: 'Cancel',
       dictDefaultMessage: 'Click here or Drop files here to upload',
-      timeout: 0
+      timeout: 0,
+      accept: acceptCallback
     };
 
+    // dropzone accept
```

Comment:
Could you elaborate on what this override is doing.

Reply:
Thanks for pointing this out. This comment has been updated to clearly indicate it is overriding the dropzone accept function to initiate multipart upload on every file upload.